### PR TITLE
remove single configuration mode

### PR DESCRIPTION
**Pull request by AlvaroVega**
https://github.com/telefonicaid/iotagent-node-lib/issues/1469

- [x] Doc

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Remove single configuration mode (#1469)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Remove: single configuration mode (along with IOTA_SINGLE_MODE env var) (#1469)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by mapedraza** [comment]
Doc remaining:

https://github.com/telefonicaid/iotagent-node-lib/blob/cea186fc1f439ca82004444992a20d48ec6d7bbc/doc/devel/architecture.md#L82-L83

**Comment by mapedraza** [comment]
Should this function be simplified?
https://github.com/telefonicaid/iotagent-node-lib/blob/cea186fc1f439ca82004444992a20d48ec6d7bbc/lib/services/devices/deviceService.js#L180-L218

**Comment by AlvaroVega** [comment]
> Doc remaining:
> 
> https://github.com/telefonicaid/iotagent-node-lib/blob/cea186fc1f439ca82004444992a20d48ec6d7bbc/doc/devel/architecture.md#L82-L83

done in https://github.com/telefonicaid/iotagent-node-lib/pull/1503/commits/27cb6c4a9080149fc228f805894f94787ab97c9f

**Comment by AlvaroVega** [comment]
> Should this function be simplified?
> 
> https://github.com/telefonicaid/iotagent-node-lib/blob/cea186fc1f439ca82004444992a20d48ec6d7bbc/lib/services/devices/deviceService.js#L180-L218

done in 
https://github.com/telefonicaid/iotagent-node-lib/pull/1503/commits/31957c84b4f6eb1d54b59bf912bee0c356cb3a2e

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**